src/Analysis/GWASV2/Shared/AttritionTableWrapper/AttritionTable/AttritionTableRow.jsx

```diff
@@ -7,16 +6,16 @@ import {
   fetchConceptStatsByHareSubset,
   fetchConceptStatsByHareSubsetCC,
   queryConfig,
-} from '../wizardEndpoints/cohortMiddlewareApi';
-import BarChart from './ChartIcons/BarChart';
-import EulerDiagram from './ChartIcons/EulerDiagram';
-import './AttritionTable.css';
+} from '../../wizardEndpoints/cohortMiddlewareApi';
+import BarChart from '../ChartIcons/BarChart';
+import EulerDiagram from '../ChartIcons/EulerDiagram';
 
 const AttritionTableRow = ({
-  cohortDefinitionId,
   otherCohortDefinitionId,
```

Comment:
let's drop `otherCohortDefinitionId`

Reply:
This has been removed, including from the useQuery call in AttritionTableRow

```
 !otherCohortDefinitionId
        ? fetchConceptStatsByHareSubset(
            cohortDefinitionId,
            currentCovariateAndCovariatesFromPrecedingRows,
            outcome,
            sourceId
          )
        : fetchConceptStatsByHareSubsetCC(
```

Reply:
Removed.